graylog2-web-interface/src/components/search/MessageTableEntry.css

```diff
@@ -0,0 +1,3 @@
+:local .timezoneInfo {
+    color: #cccccc;
```

Comment:
This is highly personal, but you can play with it and see how you like it:

I would make the icon stand a little bit more, maybe using `#aaa`, as the decorator marker. I also think the icon is too close to the timestamp, so I would add a slight `margin-left` to avoid that. This is what I used:

```
:local .timezoneInfo {
    color: #aaa;
    margin-left: 0.2em;
}
```

Before:
<img width="223" alt="screen shot 2018-01-30 at 13 45 05" src="https://user-images.githubusercontent.com/716185/35567052-e18df22c-05c3-11e8-87b0-0b9299b96755.png">

After:
<img width="237" alt="screen shot 2018-01-30 at 13 44 42" src="https://user-images.githubusercontent.com/716185/35567051-e16448d2-05c3-11e8-8eec-2943cd370498.png">

Reply:
This is of course more a suggestion than a change, so you can ignore it if you feel like it :)

Reply:
I had it before in ```#aaa``` but since you meant not to prominent I cooled it down a bit ;)
Will change it to ```#aaa``` :+1: